docs/sphinx/source/whatsnew/v0.7.0.rst

```diff
@@ -187,6 +187,10 @@ Removal of prior version deprecations
 * Removed `ModelChain.singlediode` method.
 * Removed `ModelChain.prepare_inputs` clearsky assumption when no irradiance
   data was provided.
+  
+Requirements
+~~~~~~~~~~~~
+* pvlib now requires numpy >= 1.12.0, released in 2017. (:issue:`830`)
```

Comment:
Why add this new section for numpy but not include python or pandas? I'm not opposed to the section, just looking for consistency.

Reply:
Sorry, I was just looking for examples of a section that talked about the changes in requirements, I copied this from [v0.4 requirements](https://pvlib-python.readthedocs.io/en/latest/whatsnew.html#requirements) which was the only example I could find. I only addressed numpy because that's what's in #830.

Does v0.7 also change pandas requirments? There's already a section in bold at the top which states the Python, Pandas, and NumPy requirements, but I'm happy to repeat it down here too or just remove this section if it's redundant? I don't know what's best, so if you have a strong opinion, I happy with whatever you would like to see here. Thanks!

Reply:
> Does v0.7 also change pandas requirments? There's already a section in bold at the top which states the Python, Pandas, and NumPy requirements

Yes, they've all changed which is why they are up there in bold. But it seems that we need to do a better job of making that clear. pandas puts dependency changes like these in warning boxes. I don't know if that's overdoing it or not. Maybe something like "pandas requirement increased to" would be more clear than "pvlib now requires"?

I don't have a strong opinion other than we should be consistent within this file. And the bikeshed should be white.